src/vit-testing/mainnet-tools/src/snapshot/convert.rs

```diff
@@ -130,7 +130,7 @@ impl OutputExtension for SnapshotEntry {
                     VotingDelegations::New(new)
                 }
             },
-            voting_purpose: *self.voting_purpose.unwrap_or(VotingPurpose::CATALYST),
+            voting_purpose: Some(*self.voting_purpose.unwrap_or(VotingPurpose::CATALYST)),
```

Comment:
Can we remove `unwrap_or` instead of wrapping `*self.voting_purpose` in `Some`?

Reply:
voting_purpose is an Option.  We MAY be able to remove the unwrap, but it's not clear to me what this test is even doing, so this keeps the logic as close as possible to the original, with the hope to preserve the tests functionality (whatever that is).